src/yadism/esf/exs.py

```diff
@@ -5,7 +5,67 @@
 from .esf import ESFInfo
 from .result import ESFResult, EXSResult
 
-conv = 3.893793e10  # conversion factor from GeV^-2 to 10^-38 cm^2
+GEV_CM2_CONV = 3.893793e10
+"Conversion factor from GeV^-2 to 10^-38 cm^2"
+
+
+def xs_coeffs(kind, y, x=None, Q2=None, params=None):
+    """Compute coefficients in the definition of a given cross section.
+
+    Parameters
+    ----------
+    kind : str
+        the identifier of the cross section for which coefficients have to be
+        computed
+    y : float
```

Comment:
Could you please provide some quick description? (since we're on it)

Reply:
You're right, I'll do it (as you see I started, I simply forgot to complete).